### PR TITLE
List missing builtins

### PR DESCRIPTION
**Pull request by serge-sans-paille**
except exceptions, they're another story

### CODE REVIEW

**Comment by serge-sans-paille** [comment]
@lsix ready for review!

File: penty/pentypes/builtins.py [hunk 1]
```diff
@@ -153,10 +159,18 @@ def biniop(self, value):
...
 def int_pow(self, value, mod=_Cst[None]):
     self, value = _astype(self), _astype(value)
-    if mod is not _Cst[None] and not issubclass(mod, int):
+    if not issubclass(self, int):
         raise TypeError
-    if issubclass(self, int) and issubclass(value, int):
+    if mod is not _Cst[None]:
+        if not issubclass(value, int):
+            raise TypeError
+        if not issubclass(mod, int):
```
Comment (lsix) [inline penty/pentypes/builtins.py:167]: I guess you meant `issubclass(_astype(mod), int)`

File: penty/pentypes/builtins.py [hunk 2]
```diff
@@ -703,12 +734,18 @@ def dict_values(self):
     return _DictValueIterator[self.__args__[1]]
 
 def dict_instanciate(ty):
+    keys, values = ty.__args__
+    print(ty)
```
Comment (lsix) [inline penty/pentypes/builtins.py:738]: this debugging message should be removed.

File: penty/pentypes/builtins.py [hunk 3]
```diff
@@ -703,12 +734,18 @@ def dict_values(self):
...
+    for key in keys:
+        print('hashing', key)
```
Comment (lsix) [inline penty/pentypes/builtins.py:740]: This one also

**Review by lsix — CHANGES_REQUESTED** [review]
Fast pass (during a meeting), I have not been through the tests yet.

**Pull request merged**